src/utils.ts

```diff
@@ -71,3 +73,131 @@ export const getCount = (item: number | string) => {
   const formattedItem = typeof item === "string" ? item.length : item;
   return wordsFormat.format(formattedItem);
 };
+
+export class Metadata {
+  private metaData: string;
+  public fileTitle: string;
+  public source: string;
+
+  constructor({
+    fileTitle,
+    transcript_by,
+    url,
+    date,
+    tags,
+    speakers,
+    categories,
+  }: MetadataProps) {
+    this.fileTitle = fileTitle;
+    this.source = url;
+
+    // eslint-disable-next-line prettier/prettier
+    this.metaData = `---\n` +
+                    `title: ${fileTitle}\n` +
+                    `transcript_by: ${transcript_by} \n`;
```

Comment:
```suggestion
                    `transcript_by: ${transcript_by}\n`;
```
Is this space causing an extra line to be created? see https://github.com/adamjonas/bitcointranscripts/pull/114/files#r1170717938